LL/rig_alban.trig

```diff
@@ -448,7 +450,7 @@ _:missing-02dcec05
     foaf:gender "female";
     irishRel:genName "Feidelm Fholtchain";
     irishRel:nomName "Fedelm Foltcain";
-    rel:parentOf <#Comgall>, <#Gabran-2668f767>;
+    rel:parentOf <#Domongairt-4bb754c2>;
```

Comment:
Incorrect. The source reads as follows:

> GAbran & Comgall da mc Domongairt. Et Feidelm Fholtchain ingen Briuin mc Echach Mugnedoin a m-mathair.

"Gabrán and Comgall, two sons of Domongart. And Feidelm Foltcháin, daughter of Brion, son of Eochu Mugmedón, [was] their mother."

The possessive pronoun is plural as it nasalises (maybe geminates, in this context) the following consonant.

Reply:
Heh, I looked this up in GOI and still seem to have gotten it wrong.